tests/HelpersTest.cpp

```diff
@@ -522,7 +522,11 @@ TEST_CASE("Validators: FileIsDir", "[helpers]") {
 }
 
 TEST_CASE("Validators: DirectoryExists", "[helpers]") {
+#ifdef BAZEL
```

Comment:
Is it possible to avoid having different paths here for Bazel? Bazel is running from the parent dir, while the others are running from the tests dir?

Reply:
That sounds like an elegant solution to me. I'm not quite sure why the CMake builds are failing now (I'm certainly no CMake expert)

Reply:
I made a small change to your last commit that seems to fix it

Reply:
Great, thanks!